opennms-config/src/main/java/org/opennms/netmgt/config/NotifdConfigFactory.java

```diff
@@ -86,16 +91,17 @@ static synchronized public NotifdConfigFactory getInstance() {
         return instance;
     }
 
+
+
     /**
      * <p>init</p>
      *
      * @throws java.io.IOException if any.
      * @throws java.io.FileNotFoundException if any.
      */
-    public static synchronized void init() throws IOException, FileNotFoundException {
+    public static synchronized void init() throws IOException {
         if (!initialized) {
             instance = new NotifdConfigFactory();
-            instance.reload();
```

Comment:
Moving reload out of init method as bean creation process was not completed. Due to that loading of config data is throwing an exception that results to NotifdConfigFactory bean exception